rwgame/main.cpp

```diff
@@ -9,6 +9,14 @@ int main(int argc, char* argv[])
 
 		return game.run();
 	} catch (std::runtime_error& ex) {
+		// Make sure the user can use the cursor to acknowlege the error if
+		// it occurs while the mouse is locked to the window.
+		SDL_SetRelativeMouseMode(SDL_FALSE);
+		SDL_Window* activeWindow = GameWindow::getMainWindow();
+		if (activeWindow != nullptr) {
+			SDL_SetWindowGrab(activeWindow, SDL_FALSE);
```

Comment:
Can't we just somehow force the focus away from the main window?
If it's not the same focus group it might automaticly disable mousegrab when switching focus.